contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+            assert values[values_index + 3] = Utils.WonderNames.wonder_34;
+        }
+        if (wonder == 35) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_35;
+        }
+        if (wonder == 36) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_36;
+        }
+        if (wonder == 37) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_37;
+        }
+        if (wonder == 38) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_38;
+        }
+        if (wonder == 39) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_39;
+        }
+        if (wonder == 40) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_40;
+        }
+        if (wonder == 41) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_41;
+        }
+        if (wonder == 42) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_42;
+        }
+        if (wonder == 43) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_43;
+        }
+        if (wonder == 44) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_44;
+        }
+        if (wonder == 45) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_45;
+        }
+        if (wonder == 46) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_46;
+        }
+        if (wonder == 47) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_47;
+        }
+        if (wonder == 48) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_48;
+        }
+        if (wonder == 49) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_49;
+        }
+        if (wonder == 50) {
+            assert values[values_index + 3] = Utils.WonderNames.wonder_50;
+        }
+
+        let trait_key = 639351341392214529084052888304630306;  // {"trait_type":"
+        let wonder_key = 127786802251070649640036681418183236865965976789548;  // Wonder (translated)",
+        let value_key = 635719516926804900386;  // "value":"
+        let right_bracket = Utils.Symbols.RightBracket;
+        let inverted_commas = Utils.Symbols.InvertedCommas;
+        let comma = Utils.Symbols.Comma;
+
+        assert values[values_index] = trait_key;
+        assert values[values_index + 1] = wonder_key;
+        assert values[values_index + 2] = value_key;
+        assert values[values_index + 4] = inverted_commas;
+        assert values[values_index + 5] = right_bracket;
+        assert values[values_index + 6] = comma;
+
+        return (values_index + 7,);
+    }
+    func append_felt_ascii{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        num: felt, arr: felt*
+    ) -> (added_len: felt) {
+        alloc_locals;
+        let (q, r) = unsigned_div_rem(num, 10);
+        let digit = r + 48;  // ascii
```

Comment:
Cool stuff, but took me a while to figure out how it works. Do you mind adding a short comment?